lib/expressir/liquid/declarations/schema_drop.rb

```diff
@@ -6,16 +6,52 @@ module Declarations
       class SchemaDrop < ::Expressir::Liquid::DeclarationDrop
         include ::Expressir::Liquid::IdentifierDrop
 
-        def initialize(model)
+        def initialize(model, selected_schemas: nil, options: {}) # rubocop:disable Lint/MissingSuper
```

Comment:
Why do we need the notion of “selected schemas” in Expressir? Can’t we do it in liquid?